Makefile.in

```diff
@@ -195,11 +195,16 @@ ifeq ($(shell uname), Darwin)
 endif
 
 #
-# All translation message catalogs
+# All translation message catalogs, filter files based on LINGUAS.
 #
 TRANSLATIONS_SRC := $(wildcard po/*.po)
-ifdef HAVE_GETTEXT
+ifeq ($(HAVE_GETTEXT), 1)
 	TRANSLATIONS := $(TRANSLATIONS_SRC:.po=.gmo)
+ifdef LINGUAS
```

Comment:
I had to ponder the following code for a few minutes with reference to the GNU Make docs but it looks correct. I'd like another developer like @zanchey to sign-off on this before it is merged.